### PR TITLE
Login test

### PR DESCRIPTION
**Pull request by LakshmiMadhuNair**

### CODE REVIEW

**Review by AnushiyaM11 — CHANGES_REQUESTED** [review]
Please do the needful changes

**Review by Sharz-dev — APPROVED** [review]

File: diagramming-tool/src/Component/Login/LoginPage.jsx [hunk 1]
```diff
@@ -1,19 +1,63 @@
...
+
+    if (!password.trim()) {
+      setError('Please enter your password.');
+      return;
+    }
+
+    setError('');
+    try {
+      // eslint-disable-next-line no-unused-vars
+      const response = await authApi.login(email, password);
```
Comment (AnushiyaM11) [inline diagramming-tool/src/Component/Login/LoginPage.jsx:27]: API calls are expected to be in the services folder

File: diagramming-tool/src/Component/Login/auth.js
Comment (AnushiyaM11) [inline diagramming-tool/src/Component/Login/auth.js:1]: You can move this file to services folder as it is API cal

**Review by AnushiyaM11 — APPROVED** [review]
Approved

**Review by Sharz-dev — APPROVED** [review]

**Pull request merged**